integrations/comscore/lib/index.js

```diff
@@ -89,6 +94,40 @@ Comscore.prototype.mapComscoreParams = function(page) {
     }
   });
 
+  if (this.options.consentFlag) {
```

Comment:
@pooyaj do we have a preference/standard on if we should put some of these conditions on a single line or if multiple lines are clearer?